com.microsoft.mrtk.graphicstools.unity/Samples~/MeshOutlines/Materials/OutlineOrange.mat

```diff
@@ -2,14 +2,18 @@
 %TAG !u! tag:unity3d.com,2011:
 --- !u!21 &2100000
 Material:
-  serializedVersion: 6
+  serializedVersion: 8
```

Comment:
Mind if we revert this material to the previous version? Looks like this file wasn't changed by you intentionally but was updated by Unity.